tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<ul><li>Remove <code>add, sub, mul, div ...</code> in the title. List in the text cell of <em>all</em> the supported binary element-wise operations.</li><li>Mention operads have to be </li><li class="ql-indent-1">Same device</li><li class="ql-indent-1">Same dtype</li><li class="ql-indent-1">broadcast-compatible</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>